### PR TITLE
Re-Format SVGStore

### PR DESCRIPTION
**Pull request by xerc**
formating got messed up ; manually cleaned
lines with # are intentional marked seperatly

### CODE REVIEW

**Comment by xerc** [comment]
thanks!

**Comment by lochmueller** [comment]
Hey @xerc 
Could you run "composer install && composer code" on your maschine. This will format the files with the current PHP CS fixer configuration. This will change some things back...
Regards,
Tim

**Comment by xerc** [comment]
why `<?php declare(strict_types=1);` gets mod. ?; it is as everywhere `grep -rH 'php declare' ./.Build`

**Comment by xerc** [comment]
comments at source line-endings `; //` gets modified but after `{//` or `)//` not -> does not bother me to much
but the intentionally disabled functions should be somehow marked or extracted <- you oppinion? @lochmueller

File: Resources/Public/Icons/Extension.svg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" fill="#fff">
+  <path d="M0 0h512v512H0z" fill="#fc9c04"/>
+  <path d="m206.34 168.74 110.84 140.91c27.83 35.39 18.61 18.59-1.85 43.58l-31.7 38.71c-23.14 28.27-12.61 24.4-51.07-16.24L83.33 218.06c-11.9-12.57-17.74-19.32-1.34-24.92l83.26-28.39c33.77-11.52 21.9-20.4 41.09 4z"/>
+  <path d="M419.56 101.8a21.04 21.04 0 0 0-13.51 4.47L301.99 187.5l-38.08-48.4v-.01c-11.53-14.66-35.7-14.84-54.16-.42l-10.37 8.09 143.76 182.76 10.37-8.1c18.47-14.41 24.07-37.82 12.54-52.48l-38.08-48.42 104.06-81.22a20.88 20.88 0 0 0 3.58-29.45 21.04 21.04 0 0 0-16.05-8.06z"/>
```
Comment (xerc) [inline Resources/Public/Icons/Extension.svg:4]: THX @ https://openclipart.org/artist/michelinux

**Review by xerc — COMMENTED** [review]
THX @ https://openclipart.org/artist/michelinux

**Comment by lochmueller** [comment]
Hey @xerc 

"why <?php declare(strict_types=1); gets mod. ?; it is as everywhere grep -rH 'php declare' ./.Build" Like in the core, there is a single line between the declare and the PHP opening tag:
https://git.typo3.org/typo3/typo3/-/blob/master/typo3/sysext/extbase/Classes/Annotation/IgnoreValidation.php

Related to the comments... please do not comment in source code lines. You can add your comment before the line. It is much easier to read. If there are too many comments, perhaps we could extract the lines in a single function with a good name instead using comments.

Regards,
Tim

**Comment by lochmueller** [comment]
I will merge the changes and will run the PHP CS Fixer ("composer code") later...

**Pull request merged**